lib/common.sh

```diff
@@ -25,13 +25,15 @@ setConfigFile() {
 }
 
 loadAddresses() {
+    set +x
```

Comment:
Why set it only here? Shouldn't we set it e.g. in `deploy-kovan.sh`?

Reply:
Yeah we need to evaluate how to do the printing in general. I've put it only in the `loadAddresses` to avoid the huge unnecessary output this function does.